app/javascript/Pages/components/Button.scss

```diff
@@ -0,0 +1,71 @@
+$primary-color: #6600cc;
+$dark-color: #252636;
+$white-color: #ffffff;
+$gray-color: #aaaaaa;
+
+.btn {
+  padding: 0.5rem 1.25rem;
+  border-radius: 5px;
+  outline: none;
+  border: none;
+  cursor: pointer;
+  white-space: nowrap;
+}
+
+.btn--primary {
+  background-color: var($white-color);
+  color: $primary-color;
+  border: 1px solid var($white-color);
```

Comment:
**issue (blocking)**: to use scss variables you don't need the "var" that's for CSS variables)